packages/page-accounts/src/Accounts/modals/Delegate.tsx

```diff
@@ -31,10 +31,15 @@ function Delegate ({ amount: _amount, conviction: _conviction, delegatedAccount:
   const [delegatedAccount, setDelegatedAccount] = useState<string | null>(_delegatedAccount || null);
   const [conviction, setConviction] = useState(_conviction?.toNumber() || 1);
 
+  const isDirty = amount?.toString() !== _amount?.toString() ||
+    delegatedAccount !== _delegatedAccount ||
+    delegatingAccount !== _delegatingAccount ||
+    conviction !== _conviction?.toNumber();
```

Comment:
I find this incredibly hard to read for a couple of reasons -

- we don't do the props to `_` casting anywhere else in the codebase
- not sure (just glancing) which is which
- at later points we mix the _/non_ (e.g. in label)

(More of a nitpick.)

Small issue spotted with conviction override as well, when set to "none" it will always reset here to "0". Not introduced here, but introduced in the previous.

Reply:
Agreed, that `_*` naming is not great at all. I'll rename for `previous*`, it should be better.